provisioner/powershell/provisioner.go

```diff
@@ -116,15 +116,15 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 	}
 
 	if p.config.ElevatedEnvVarFormat == "" {
-		p.config.ElevatedEnvVarFormat = `$env:%s="%s"; `
+		p.config.ElevatedEnvVarFormat = `$env:%s=\"%s\"; `
 	}
 
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = `powershell "& { {{.Vars}}{{.Path}}; exit $LastExitCode}"`
+		p.config.ExecuteCommand = `powershell '& {$ProgressPreference=\"SilentlyContinue\"; {{.Vars}}{{.Path}}; exit $LastExitCode}'`
```

Comment:
Ok, good to know. I hadn't problems building Win10/2016 for VirtualBox and Fusion/Workstation with standard settings of packer 0.10.